From 94a2af7dcb88d4997cdfeb359cb9c63c4e05d255 Mon Sep 17 00:00:00 2001
From: Vitaly Andrianov <vitalya@ti.com>
Date: Mon, 20 Feb 2017 12:37:55 -0500
Subject: [PATCH 92/94] am57: jailhouse: disable using timer7/8 and uart9 by
 Linux

In order to use some peripheras in a jailhouse inmate we need to
let Linux know about that. Linux still has to take care about their
clockes, but power manager doesn't have to touch them.

So, we tell Linux not to use timer7, timer8 and uart9

Signed-off-by: Vitaly Andrianov <vitalya@ti.com>
Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 arch/arm/boot/dts/am57xx-evm-reva3.dts | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/arch/arm/boot/dts/am57xx-evm-reva3.dts b/arch/arm/boot/dts/am57xx-evm-reva3.dts
index 28c90c0..484bc61 100644
--- a/arch/arm/boot/dts/am57xx-evm-reva3.dts
+++ b/arch/arm/boot/dts/am57xx-evm-reva3.dts
@@ -28,6 +28,21 @@
 	};
 };
 
+&timer8 {
+	status = "disabled";
+	ti,no-idle;
+};
+
+&timer7 {
+	status = "disabled";
+	ti,no-idle;
+};
+
+&uart9 {
+	status = "disabled";
+	ti,no-idle;
+};
+
 &dra7_pmx_core {
 
 	/*
-- 
1.9.1

